package.json

```diff
@@ -19,24 +19,24 @@
   },
```

Comment:
These were bumped to their latest. None of them appears to have broken interface.